sp800-63d/vot_mapping.md

```diff
@@ -1,54 +1,82 @@
-<a name="sec1"></a>
+<a name="vectors-of-trust-mapping"></a>
 
 <div class="breaker"></div>
 
-## VOT Overview
+## Mapping Assurance Levels to Vectors of Trust
+_This section is normative._
 
-VOT is comprised of components, and components contain values.
+Vectors of Trust (VoT) is a standard for expressing information about an authentication transaction from an IdP to an RP. This information is split into several orthogonal component categories, much in the same way that the assurances defined in 800-63 are split into different xAL categories.
 
-## Assurance to VOT Mapping
+### Trustmark
 
-_This section is normative._
+The trustmark URI for these definitions is [[ URI for this document ]].
+
+### xAL to VoT Components
+
+This document uses three categories defined in VoT: Identity Proofing (P), Primary Credential Usage (C), and Assertion Presentation (A). These categories correspond to the Identity Assurance Level (IAL), Authenticator Assurance Level (AAL), and Federation Assurance Level (FAL), respectively. 
+
+This document does not use the Primary Credential Management (M) category defined in VoT.
+
+This document does not define any additional component categories. Any additional categories SHALL NOT be used.
 
-### xAL to VOT Components
+## Component Values
 
-|xAL|Component|
-|:--:|:----|
-|IAL|Identity Proofing|
-|AAL Authenticator Requirements|Primary Credential Usage|
-|AAL Authenticator Lifecycle|Primary Credential Management|
-|FAL|Assertion Presentation|
+This document does not use any of the existing values in VoT but instead defines new component values here in accordance with Section 8 of VoT.
 
 ### IAL to VOT Component Values
 
-May need to break some of these up by remote, remote supervised, in-person.
+IAL maps into the VoT component P using a numeric identifier. Only one numeric identifier SHALL be used. 
 
-IAL maps into the VOT component
+|IAL|Component Value|
+|:----:|:--:|
+|IAL1 (with no attributes)|P0|
+|IAL1|P1|
+|IAL2|P2|
+|IAL3|P3|
 
-|IAL|Options|Component Value|
-|:----:|:---|:--:|
-|IAL1|No attributes|P0|
-|IAL1|Self-asserted attributes|P1|
-|IAL2|Unsupervised Remote and In-Person|P2|
-|IAL3|Supervised Remote or In-person|P3|
+In addition, the type of proofing used MAY be indicated using an alphabetic identifier. Multiple alphabetic identifiers MAY be used simultaneously.
 
+|Proofing method|Component Value|
```

Comment:
I'm not sure we really need to expose 'sub-sub methods since we deemed them equivalent, but let's consider a vector for:
remote - 1 form of id (there is one case where this is allowed)
in person: supervised remote

And we should have an assertion for 'we've chosen based on risk to be IALx but not implement the requirements'. And maybe a URI to the document that states the compensating controls???